### PR TITLE
make date human readable

### PR DESCRIPTION
**Pull request by tamene21**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
   "author": "Lucas Braun",
   "license": "MIT",
   "dependencies": {
-    "angular": "^1.6.8",
+    "angular": "^1.6.9",
```
Comment (lbraun) [inline package.json]: Why are you changing the angular version?

File: js/index.js [hunk 2]
```diff
@@ -63,9 +63,11 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
 
   function loadTodos(response) {
     if (response.data.length > 0) {
+      var dateString;
       $scope.markers = response.data;
       $($scope.markers).each(function(index, marker) {
-        $scope.markers[index].message = "<b>Title: </b>" + marker.title + "<br/><b>Due Date: </b><b>" + marker.dueDate + "</b>";
+        dateString= new Date(marker.dueDate).toGMTString();
```
Comment (lbraun) [inline js/index.js]: Missing a space

File: js/index.js [hunk 3]
```diff
@@ -63,9 +63,11 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
...
+        $scope.markers[index].message = "<b>Title: </b>" + marker.title + "<br/><b>Due Date: </b>" + dateString.substring(0, dateString.length - 4);
```
Comment (lbraun) [inline js/index.js]: Can you move the substring() operation to the previous line to make this line less long?

**Review by lbraun — COMMENTED** [review]
Looks good dude 😎

File: package.json (hunk 1 above)
Comment (tamene21) [reply package.json]: I issued npm install angular by mistake instead of just npm install.  use the original one

**Comment by lbraun** [comment]
@Tamenegelaye why did you delete this?

**Comment by lbraun** [comment]
Nevermind 😄 I see you made #20 instead

**Pull request closed without merging**